server/grpc/api/service.go

```diff
@@ -11,19 +12,21 @@ import (
 // ServiceServer is the type to aggregate all Service APIs.
 type ServiceServer struct {
 	sdk *sdk.SDK
+	cfg *config.Config
 }
 
 // NewServiceServer creates a new ServiceServer.
-func NewServiceServer(sdk *sdk.SDK) *ServiceServer {
-	return &ServiceServer{sdk: sdk}
+func NewServiceServer(sdk *sdk.SDK, cfg *config.Config) *ServiceServer {
+	return &ServiceServer{sdk: sdk, cfg: cfg}
 }
 
 // Create creates a new service from definition.
 func (s *ServiceServer) Create(ctx context.Context, req *protobuf_api.CreateServiceRequest) (*protobuf_api.CreateServiceResponse, error) {
-	credUsername, credPassphrase, err := GetCredentialFromContext(ctx)
-	if err != nil {
-		return nil, err
-	}
+	// credUsername, credPassphrase, err := GetCredentialFromContext(ctx)
+	// if err != nil {
+	// 	return nil, err
+	// }
+	credUsername, credPassphrase := s.cfg.Account.Name, s.cfg.Account.Password
```

Comment:
Hum, what? Do we have only one user here? I don't know what purpose is this for, but it looks like step back.

Reply:
Yes, only one account will be used until we implement a full account management with role or user authorization.
See comment in issue <https://github.com/mesg-foundation/engine/issues/1458>.